include/envoy/request_info/request_info.h

```diff
@@ -68,66 +68,125 @@ class RequestInfo {
    */
   virtual void onUpstreamHostSelected(Upstream::HostDescriptionConstSharedPtr host) PURE;
 
+  /**
+   * @return the # of body bytes received in the request.
```

Comment:
super nitty: would prefer number be spelled out

Reply:
Yep, that was an existing comment just mangled in the diff. Will fix for both received and sent.